mpp/include/mpp_domains_misc.inc

```diff
@@ -512,7 +512,7 @@ end subroutine init_nonblock_type
       logical :: native         !true if I'm on the pelist of this domain
       integer :: listsize, listpos
       integer :: n
-      integer, dimension(11) :: msg, info         !pe and compute domain of each item in list
+      integer, dimension(12) :: msg, info         !pe and compute domain of each item in list
```

Comment:
I do not like this `12` (previously 11) to be coded like this.  It does not allow future developers to know why the dimension of these arrays is specifically 12.